From d11adca61b022d66bd0540e9a7a8da9e118fa221 Mon Sep 17 00:00:00 2001
From: Amrathesh <amrathesh@arm.com>
Date: Tue, 21 Jun 2022 22:14:28 +0530
Subject: [PATCH] Fix for CompuLab IOT-GATE-iMX8 boot issue

- USB doesn't get mounted due to power domain support
  changes for SoC.

Signed-off-by: Amrathesh <amrathesh@arm.com>
---
 drivers/soc/imx/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/imx/Makefile b/drivers/soc/imx/Makefile
index 078dc918f..fbdadcac7 100644
--- a/drivers/soc/imx/Makefile
+++ b/drivers/soc/imx/Makefile
@@ -3,5 +3,5 @@ ifeq ($(CONFIG_ARM),y)
 obj-$(CONFIG_ARCH_MXC) += soc-imx.o
 endif
 obj-$(CONFIG_HAVE_IMX_GPC) += gpc.o
-obj-$(CONFIG_IMX_GPCV2_PM_DOMAINS) += gpcv2.o
+#obj-$(CONFIG_IMX_GPCV2_PM_DOMAINS) += gpcv2.o
 obj-$(CONFIG_SOC_IMX8M) += soc-imx8m.o
-- 
2.17.1

